string pcd_path
geometry_msgs/PoseStamped init_pose